client/src/pages/boxes/BoxesPage.module.less

```diff
@@ -11,3 +11,107 @@
 .table_row {
   background-color: #fff0f6;
 }
+
+.page4 h3 {
+  font-size: 1.5rem;
+  margin: 1rem;
+}
+
+.contentTable {
```

Comment:
Are these styles shared among other components? if not, would be a good idea to move them to separate file